.github/workflows/release.yaml

```diff
@@ -0,0 +1,41 @@
+name: Release
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  github:
+    name: Make a github release with GoReleaser
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v2
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          version: latest
+          args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  docker-hub:
+    name: Deploy to docker-hub
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Login to docker hub
+        uses: actions-hub/docker/login@master
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_TOKEN }}
+
+      - name: Build image
+        if: success()
+        run: docker build -t ${GITHUB_REPOSITORY}/cli:${IMAGE_TAG} .
```

Comment:
If I not wrong you need to declare the $IMAGE_TAG variable, isn't it?

Reply:
Yeap, it looks like I forgot it.

Reply:
Nope, I forgot again. This [docker cli](https://github.com/actions-hub/docker/blob/master/cli/entrypoint.sh) create an env variable called IMAGE_TAG and IMAGE_NAME (which are GITHUB_REPOSITORY). So this env is already there.